### ISSUE TITLE
Error when compile to production

### ISSUE THREAD

**Issue by viruzck** [jrquick17/ionic4-auto-complete#42]
# I'm submitting a

- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [X] Bug report
- [ ] Enhancement
- [ ] Feature request
- [ ] Documentation issue or request

## Description

Hi i have a error when run "ionic cordova build android --prod", in my angular.json file i have @ionic/angular-toolkit 

```
ERROR in ./node_modules/ionic4-auto-complete/auto-complete/auto-complete.component.ts
Module build failed (from ./node_modules/@ngtools/webpack/src/index.js):
Error: D:\Erick\Git\XXXXX\node_modules\ionic4-auto-complete\auto-complete\auto-complete.component.ts is missing from the TypeScript compilation. Please make sure it is in your tsconfig via the 'files' or 'include' property.
The missing file seems to be part of a third party library. TS files in published libraries are often a sign of a badly packaged library. Please open an issue in the library repository to alert its author and ask them to package the library using the
 Angular Package Format (https://goo.gl/jB3GVv).
    at AngularCompilerPlugin.getCompiledFile (D:\Erick\Git\XXXXX\node_modules\@ngtools\webpack\src\angular_compiler_plugin.js:752:23)
    at plugin.done.then (D:\Erick\Git\XXXXX\node_modules\@ngtools\webpack\src\loader.js:41:31)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
ERROR in ./node_modules/ionic4-auto-complete/bold-prefix.pipe.ts
Module build failed (from ./node_modules/@ngtools/webpack/src/index.js):
Error: D:\Erick\Git\XXXXX\node_modules\ionic4-auto-complete\bold-prefix.pipe.ts is missing from the TypeScript compilation. Please make sure it is in your tsconfig via the 'files' or 'include' property.
The missing file seems to be part of a third party library. TS files in published libraries are often a sign of a badly packaged library. Please open an issue in the library repository to alert its author and ask them to package the library using the
 Angular Package Format (https://goo.gl/jB3GVv).
    at AngularCompilerPlugin.getCompiledFile (D:\Erick\Git\XXXXX\node_modules\@ngtools\webpack\src\angular_compiler_plugin.js:752:23)
    at plugin.done.then (D:\Erick\Git\XXXXX\node_modules\@ngtools\webpack\src\loader.js:41:31)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
```

### Version Information

```
Ionic:

   ionic (Ionic CLI)             : 4.8.0 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.6.2
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.2.4
   @angular/cli                  : 7.2.4
   @ionic/angular-toolkit        : 1.3.0

Cordova:

   cordova (Cordova CLI) : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms     : browser 6.0.0, ios 5.1.1
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.1.3, (and 13 other plugins)

System:

   ios-deploy : 1.9.4
   ios-sim    : 8.0.2
   NodeJS     : v8.15.0 (/usr/local/bin/node)
   npm        : 6.4.1
   OS         : macOS Mojave  / Or Windows 10
   Xcode      : Xcode 11.3 Build version 11C29
```

**Comment by solucionescol** [comment]
+1

**Comment by PrithvirajChauhan** [comment]
+1

**Comment by jrquick17** [comment]
If anyone can create a reproduction of the error and share it then that would be great. I have been trying with [this temporary repository](https://github.com/jrquick17/i4ac-test) but have not had gotten the same error yet.

**Comment by solucionescol** [comment]
the issue is in version 1.9.0 and angular 7

**Comment by solucionescol** [comment]
> If anyone can create a reproduction of the error and share it then that would be great. I have been trying with [this temporary repository](https://github.com/jrquick17/i4ac-test) but have not gotten the same error yet.

the issue is in version 1.9.0 and angular 7

**Comment by jrquick17** [comment]
I've finally reproduced it [here](https://github.com/jrquick17/i4ac-test/tree/ng7) and it has been fixed [here](https://github.com/jrquick17/ionic4-auto-complete/tree/v1.10.0). However, there are still a few items that need to be completed before it can be released. It will be released as v1.10.0 when complete.

**Comment by jrquick17** [comment]
[v1.10.0 has been released](https://github.com/jrquick17/ionic4-auto-complete/releases/tag/v1.10.0) and it fixes this build error along with adding some of the newer features used in v2.5.1.
